.gitignore

```diff
@@ -3,7 +3,7 @@ build/
 gen/
 .gradle
 local.properties
-
+app/app.iml
```

Comment:
Can go under *.iml below. Also please delete the existing app/app.iml file from git.